packages/paypal-js/types/script-options.d.ts

```diff
@@ -21,6 +21,7 @@ interface PayPalScriptQueryParameters {
 }
 
 interface PayPalScriptDataAttributes {
+    env?: "sandbox" | "production";
```

Comment:
Would you be open to renaming to `environment`? I try to avoid abbreviations whenever possible. 
```suggestion
    environment?: "sandbox" | "production";
```

Reply:
Yes sounds good to me ✅ .  I have also moved the type to the PayPalScriptOptions type